kubernetes/celery-beat/deployment.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: celery-beat-deployment
+  labels:
+    app: celery-beat
+spec:
+  replicas: {{.Values.celeryBeat.replicaCount}}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: celery-beat
+  template:
+    metadata:
+      labels:
+        app: celery-beat
+    spec:
+      containers:
+      - name: celery-beat
+        image: {{.Values.django.image.repo}}:{{.Values.django.image.tag}}
+        command: [ "/entrypoint", "/start-celerybeat"]
+        envFrom:
+        - secretRef:
+            name: django-env-secrets
+        - secretRef:
+            # POSTGRES_USER and POSTGRES_PASSWORD
+            name: postgres-env-secrets
+        env:
+        - name: POSTGRES_HOST
+          value: {{.Values.database.dbIpAddressWithinContainer}}
+        - name: POSTGRES_DB
+          value: {{.Values.database.dbName}}
+        - name: POSTGRES_PORT
+          value: {{.Values.database.dbPort}}
+        volumeMounts:
+        - name: django-fileserver-storage
+          mountPath: /data
+      - name: cloud-sql-proxy
+        image: {{.Values.database.cloudsqlimage}}
+        command:
+        - "/cloud_sql_proxy"
+        - "-ip_address_types=PRIVATE"
+        - "--instances={{.Values.database.instanceConnectionName}}=tcp:{{.Values.database.dbPort}}"
```

Comment:
[Importance 2/10]

Idealy we should use configmap + env variable

Reply:
I can look at it ?

Reply:
I did it, thanks.